### DISCUSSION TITLE
Remove comments from sourcemaps

### DISCUSSION THREAD

**Discussion by dontsave** [vercel/next.js#28927] [Ideas]
### Describe the feature you'd like to request

It should be possible to remove comments from sourcemaps as they potentially expose sensitive information about the project. Currently babel in Next removes comments from generated js files, but sourcemaps still maintain the comments, posing a security concern.

### Describe the solution you'd like

a setting in next.config

### Describe alternatives you've considered

setting `comments: false` in babelrc does not solve the issue

**Comment by BlakePetersen** [comment]
FWIW, this issue isn't specific to Next.js as I am currently looking to resolve this same issue but with CRA.

**Comment by votemike** [comment]
If we are now using SWC instead of Babel... do we need to write a SWC plugin?

**Comment by pzi** [comment]
I would also love to know how to remove comments from sourcemaps, so here's my +1.
